src/track/serato/tags.cpp

```diff
@@ -129,38 +133,34 @@ double SeratoTags::findTimingOffsetMillis(const QString& filePath) {
         case EXIT_CODE_CASE_D:
             timingOffset = -60;
             break;
+        }
 #elif defined(__MAD__) || defined(__FFMPEG__)
-        // Apparently all mp3guessenc cases have the same offset for MAD
-        // and FFMPEG
-        default:
-            timingOffset = -19;
-            break;
-#endif
+        if (signalInfo.getSampleRate() == 48000) {
+            timingOffset = -24;
+        } else if (signalInfo.getSampleRate() == 44000) {
+            timingOffset = -26;
         }
```

Comment:
add an else branch to warn that we don't know the timing offset for other sample rates